lib/console/etl_worker.ex

```diff
@@ -0,0 +1,305 @@
+defmodule Console.EtlWorker do
+  use GenServer
+  alias Console.Repo
+  alias Console.Devices
+  alias Console.Channels
+  alias Console.Organizations
+  alias Console.AlertEvents
+
+  def start_link(initial_state) do
+    GenServer.start_link(__MODULE__, initial_state, name: __MODULE__)
+  end
+
+  def init(_opts) do
+    schedule_events_etl(100)
+    {:ok, %{}}
+  end
+
+  def handle_info(:run_events_etl, state) do
+    Task.Supervisor.async_nolink(ConsoleWeb.TaskSupervisor, fn ->
```

Comment:
we're using a task for this so it runs async w/ other jobs like sending emails. multiple `handle_info` won't be running at once, right?

Reply:
only 1 handle info will run at a time.

running a task here so that if it fails, it doesnt take down the entire etl_worker genserver.